### ISSUE TITLE
Remove dependency on inflight, switch to lru-cache?

### ISSUE THREAD

**Issue by robogeek** [YousefED/typescript-json-schema#608]
I am getting this warning:

```shell
$ npm install
npm WARN deprecated inflight@1.0.6: This module is not supported, and leaks memory. Do not use it. Check out lru-cache if you want a good and tested way to coalesce async requests by a key value, which is much more comprehensive and powerful.
```

My application does not directly depend on inflight, instead it comes from this module:

```shell
$ npm ls inflight
esx-core-server@1.0.0 /home/david/Projects/Evoke-mac/esx-core/server
└─┬ quicktype@23.0.170
  └─┬ quicktype-typescript-input@23.0.170
    └─┬ @mark.probst/typescript-json-schema@0.55.0
      └─┬ glob@7.2.3
        └── inflight@1.0.6
```

Further discussion https://github.com/YousefED/typescript-json-schema/issues/605
